action.yml

```diff
@@ -5,6 +5,9 @@ branding:
   color: blue
   icon: anchor
 inputs:
+  verify_blob:
+    description: "determines whether the download blob should be verified (default: true)"
+    default: 'true'
```

Comment:
if you disable that, i think you should also not install the cosign actions

Reply:
You are absolutely right - I fixed that.